### ISSUE TITLE
Server crash with 5.3.0

### ISSUE THREAD

**Issue by pkfortis** [Delocuro/Minestrapp-1.12#20]
Based on the crash report, it looks like Minestrapp is attempting to call a client-only class:

```
Time: 5/23/18 3:04 AM
Description: Exception in server tick loop
 
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Minestrappolation 5 (minestrapp)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
```

This crash doesn't occur with 5.2.1A.

Full crash report: https://pastebin.com/pzRFbub9

**Comment by Wolframme** [comment]
I can second this, I am getting the same issue with launching my server with 5.3.0.

**Comment by Delocuro** [comment]
Yep, known issue -- found out about that the day after 5.3.0 dropped. Should be fixed in 5.3.1, which'll be released in a few minutes.
